workers/datasets_based/src/datasets_based/workers/first_rows.py

```diff
@@ -521,7 +532,10 @@ def compute_first_rows_response(
 
     surrounding_json_size = get_json_size(response_features_only)
     if surrounding_json_size > rows_max_bytes:
-        raise TooBigContentError("The first rows content after truncation exceeds the maximum size.")
+        raise TooBigContentError(
+            f"The size of the first rows content after truncation ({surrounding_json_size} B) exceeds the maximum"
+            f" supported size ({rows_max_bytes} B)."
+        )
 
```

Comment:
The same, should we suggest to reduce the size of the rows?

Reply:
Hmmm, in that case, it's an issue with our configuration (see https://github.com/huggingface/datasets-server/pull/749#discussion_r1095587218), and the details will not be shown to the user, just the error message.

You're right, it should be clear that the user has nothing special to do. Note that the viewer will automatically propose to create an issue here.